s3/aide.go

```diff
@@ -26,6 +28,15 @@ type Service struct {
 	svc s3iface.S3API
 }
 
+type ReaderService interface {
+	Read(string) (*io.ReadCloser, error)
+}
+
+type Object struct {
+	Key string
+	Svc ReaderService
+}
```

Comment:
This type is exported and therefore requires documentation.

Reply:
It does. Will address.